README.md

```diff
@@ -177,7 +177,26 @@ Currently, the following clients are available for use:
     client enabled, age-restricted tracks are **not** guaranteed to play.
 
 ## Migration from Lavaplayer's built-in YouTube source
-This client is intended to be a drop-in replacement, however there are a couple of things to note:
+
+This client is intended as a direct replacement for Lavaplayer's `YoutubeAudioSourceManager`, which the maintainers
+deprecated in release [2.1.2](https://github.com/lavalink-devs/lavaplayer/releases/tag/2.1.2).
+By default, Lavaplayer instantiates and registers an instance of its deprecated `YoutubeAudioSourceManager` unless 
+explicitly excluded.
+
+To avoid this unwanted instantiation, leverage the `excludeSources` parameter during remote source registration.
+```java
+AudioPlayerManager playerManager = new DefaultAudioPlayerManager();
+AudioSourceManagers.registerRemoteSources(playerManager,
+                                          com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager.class);
+```
+
+Then, create and register an instance of the supported `YoutubeAudioSourceManager` from the `youtube-source` package.
```

Comment:
Move this section above the first codeblock. The new source manager should be registered before the others, as `registerRemoteSources` includes the `HttpAudioSourceManager` which catches all identifiers passed to it, meaning that the youtube source would never get queried when used like this.